lib/discordgo/components.go

```diff
@@ -121,7 +126,7 @@ const (
 // ComponentEmoji represents button emoji, if it does have one.
 type ComponentEmoji struct {
 	Name     string `json:"name,omitempty"`
-	ID       int64  `json:"id,string,omitempty"`
+	ID       string `json:"id,omitempty"`
```

Comment:
All YAG's snowflake IDs are int64, why revert back to string?

Reply:
I'm not sure, this was an upstream bwmarrin/discordgo update. I copy-pasted the entire file from upstream, I should have looked more carefully at each change.

Reply:
Bwmarrin's are all strings, YAG uses int64